### PR TITLE
Add first wave of Track 2 JS management packages to CSV

### PR DESCRIPTION
**Pull request by nickzhums**

### CODE REVIEW

**Review by lirenhe — APPROVED** [review]

File: _data/releases/latest/js-packages.csv [hunk 1]
```diff
@@ -432,14 +432,14 @@
...
-"@azure/arm-netapp","14.1.0","","Resource Management - NetApp","NetApp Files","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/netapp/arm-netapp","NA","NA","mgmt","false","","","","","",""
-"@azure/arm-network","25.1.0","","Resource Management - Network","Virtual Network","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/network/arm-network","https://docs.microsoft.com/javascript/api/overview/azure/virtual-network","NA","mgmt","false","","","","","",""
+"@azure/arm-netapp","14.1.0","","Resource Management - NetApp","NetApp Files","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/netapp/arm-netapp","NA","NA","mgmt","true","","","","","",""
+"@azure/arm-network","25.1.0","30.0.0-beta.1","Resource Management - Network","Virtual Network","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/network/arm-network","https://docs.microsoft.com/javascript/api/overview/azure/virtual-network","NA","mgmt","false","","","","","",""
```
Comment (qiaozha) [inline _data/releases/latest/js-packages.csv:436]:
```suggestion
"@azure/arm-network","25.1.0","30.0.0-beta.1","Resource Management - Network","Virtual Network","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/network/arm-network","https://docs.microsoft.com/javascript/api/overview/azure/virtual-network","NA","mgmt","true","","","","","",""
```

File: _data/releases/latest/js-packages.csv [hunk 2]
```diff
@@ -432,14 +432,14 @@
...
 "@azure/arm-monitor-profile-2019-03-01-hybrid","1.1.0","","Resource Management - Monitor","Monitor","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/monitor/arm-monitor-profile-2019-03-01-hybrid","NA","NA","mgmt","false","","","","true","",""
 "@azure/arm-mysql","4.0.0","","Resource Management - MySQL","Database for MySQL","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/mysql/arm-mysql","https://docs.microsoft.com/javascript/api/overview/azure/mysql","NA","mgmt","false","","","","","",""
-"@azure/arm-netapp","14.1.0","","Resource Management - NetApp","NetApp Files","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/netapp/arm-netapp","NA","NA","mgmt","false","","","","","",""
-"@azure/arm-network","25.1.0","","Resource Management - Network","Virtual Network","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/network/arm-network","https://docs.microsoft.com/javascript/api/overview/azure/virtual-network","NA","mgmt","false","","","","","",""
+"@azure/arm-netapp","14.1.0","","Resource Management - NetApp","NetApp Files","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/netapp/arm-netapp","NA","NA","mgmt","true","","","","","",""
```
Comment (qiaozha) [inline _data/releases/latest/js-packages.csv:435]:
```suggestion
"@azure/arm-netapp","14.1.0","","Resource Management - NetApp","NetApp Files","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/netapp/arm-netapp","NA","NA","mgmt","false","","","","","",""
```

**Review by qiaozha — APPROVED** [review]

File: _data/releases/latest/js-packages.csv [hunk 3]
```diff
@@ -373,7 +373,7 @@
 "@azure/arm-commerce","2.2.0","","Resource Management - Commerce","Commerce","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/commerce/arm-commerce","https://docs.microsoft.com/javascript/api/overview/azure/commerce","NA","mgmt","false","","","","","",""
 "@azure/arm-commitmentplans","1.3.0","","Resource Management - Commitment Plans","Machine Learning","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/machinelearning/arm-commitmentplans","NA","NA","mgmt","false","","","","","",""
 "@azure/arm-communication","2.1.0","","Resource Management - Communication","Communication","NA","NA","NA","mgmt","false","","","","","",""
-"@azure/arm-compute","16.2.0","","Resource Management - Compute","Compute","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/compute/arm-compute","https://docs.microsoft.com/javascript/api/overview/azure/virtualmachines","NA","mgmt","false","","","","","",""
+"@azure/arm-compute","16.2.0","30.0.0-beta.1","Resource Management - Compute","Compute","https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/compute/arm-compute","https://docs.microsoft.com/javascript/api/overview/azure/virtualmachines","NA","mgmt","true","","","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/js-packages.csv:376]: Do you know why you had to add these versions manually? I would have expected my automation to includes them.

**Review by weshaggard — APPROVED** [review]
The changes look fine as is but I will investigate why some of the versions didn't update correctly and we will also want to get these automatically detected for the next waves.

**Comment by nickzhums** [comment]
@weshaggard thanks!

**Pull request merged**